ipa-core/src/secret_sharing/vector/transpose.rs

```diff
@@ -768,6 +789,83 @@ impl_transpose_shares_ba_to_bool_small!(BA8, 256, 8, test_transpose_shares_ba_to
 impl_transpose_shares_ba_to_bool_small!(BA5, 256, 5, test_transpose_shares_ba_to_bool_256x5);
 impl_transpose_shares_ba_to_bool_small!(BA3, 256, 3, test_transpose_shares_ba_to_bool_256x3);
 
+// Special transpose used for "aggregation intermediate". See [`aggregate_contributions`] for details.
+macro_rules! impl_aggregation_transpose {
```

Comment:
Maybe we can add a test that just tests the transpose functionality here - I worry about only having coverage for it through the aggregation